# Goal
int32 axis_number         # the axis to which the request is directed
float64 distance		#desired distance to push the wire (mm)
float64 velocity		#desired velocity (mm/min)
float64 acceleration		#desired acceleration (mm/s^2)
---
# Result
bool success			# whether the operation was successful
string message			# Optional result message
---
# Feedback
float64 current_distance	# Progress of the wire feed (mm)
float64 percent			# Progress of the wire feed (%)
float64 torque_feedback		# Current motor torque (in-oz)
